WARNING:root:simply throwing exception
WARNING:root:simply throwing exception
WARNING:root:simply throwing exception
ERROR:root:simply throwing exception
NoneType: None
ERROR:root:simply throwing exception
Traceback (most recent call last):
  File "test.py", line 11, in <module>
    raise CustomException("simply throwing exception")
CustomException: simply throwing exception
